src/IdentityModel/Client/TokenClient.cs

```diff
@@ -65,27 +65,17 @@ public TokenClient(string address, HttpMessageHandler innerHttpMessageHandler)
         /// <param name="clientId">The client identifier.</param>
         /// <param name="style">The authentication style.</param>
         public TokenClient(string address, string clientId, AuthenticationStyle style = AuthenticationStyle.PostValues)
-            : this(address, clientId, string.Empty, new HttpClientHandler(), style)
+            : this(address, clientId, string.Empty, style: style)
         { }
 
-        /// <summary>
-        /// Initializes a new instance of the <see cref="TokenClient"/> class.
-        /// </summary>
-        /// <param name="address">The address.</param>
-        /// <param name="clientId">The client identifier.</param>
-        /// <param name="clientSecret">The client secret.</param>
-        /// <param name="style">The authentication style.</param>
-        public TokenClient(string address, string clientId, string clientSecret, AuthenticationStyle style = AuthenticationStyle.BasicAuthentication)
-            : this(address, clientId, clientSecret, new HttpClientHandler(), style)
```

Comment:
This ctor was no longer needed as there is an overload with optional `innerHttpMessageHandler` and `style` (whose default value is `AuthenticationStyle.BasicAuthentication`)